test/e2e/requirements.txt

```diff
@@ -0,0 +1,34 @@
+acktest @ git+https://github.com/RedbackThomson/ack-test-infra.git@02f15b6179f56383ec8026a9aa9124b55b988145
```

Comment:
refers to your personal GH repo :)

Reply:
Yep. All service controllers subject to one last commit where I update this to the HEAD of the `test-infra` once it's merged :)

Reply:
k. that is merged now :)